storage/innobase/xtrabackup/doc/source/installation/compiling_xtrabackup.rst

```diff
@@ -31,37 +31,62 @@ In Debian-based distributions, you need to:
 
 .. code-block:: bash
 
- $ apt-get install build-essential flex bison automake autoconf \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
-    libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
+   $ apt install build-essential flex bison automake autoconf \
+   libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev \
+   libgcrypt11-dev libev-dev libcurl4-gnutls-dev vim-common
 
-
-In ``RPM``-based distributions, you need to:
+In ``RPM``-based distributions, you need to install and enable `the
+Developer Toolset 7
+<https://www.softwarecollections.org/en/scls/rhscl/devtoolset-7/>`_ to
+make sure that you use the latest GCC compiler and development tools. |Percona Xtrabackup|
+requires GCC version 5.3 or higher. Then, install ``cmake3`` and other dependencies:
 
 .. code-block:: bash
 
-  $ yum install cmake gcc gcc-c++ libaio libaio-devel automake autoconf \
-    bison libtool ncurses-devel libgcrypt-devel libev-devel libcurl-devel \
-    vim-common
+   $ yum install cmake3 openssl-devel libaio libaio-devel automake autoconf \
+   bison libtool ncurses-devel libgcrypt-devel libev-devel libcurl-devel zlib-devel \
+   vim-common
+
+.. important::
+
+   In order to build |Percona XtraBackup| v8.0 from source, you need
+   to use `cmake` version 3. In your distribution it may be available
+   either as a separate package ``cmake3`` or as ``cmake``. To check,
+   run ``cmake --version`` and if it does report a version 3, install
+   ``cmake3`` for your system.
 
 Compiling with CMake
---------------------
+--------------------------------------------------------------------------------
 
-At the base directory of the source code tree, if you execute:
+At the base directory of the source code tree, run `cmake` or
+`cmake3` as follows:
 
 .. code-block:: bash
 
-  $ cmake -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF && make -j4
+   $ cmake3 -DWITH_BOOST=PATH-TO-BOOST-LIBRARY -DDOWNLOAD_BOOST=ON \
```

Comment:
I think it is better to keep `cmake` in examples, and make a clearly visible note that if cmake3 was package installed, then cmake3 command should be used.